static/sw.js

```diff
@@ -0,0 +1,11 @@
+importScripts('/bower_components/sw-toolbox/sw-toolbox.js');
+
+toolbox.options.debug = true;
+toolbox.router.default = toolbox.fastest;
+
+const filesToCache = [
+    '/',
+    'https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&lang=en'
```

Comment:
Good point - I was desperately looking for some kind of external resource to cache. It doesn't make sense to explicitly precache it anyway as the font will be loaded (and cached) together with every page.

Regarding caching other resources, I plan to:

* precache samples + supporting files (img, json resources). 
* precache AMP components (which don't have CORS enabled, so I'm not sure yet if that's a good idea)